storage-proofs-update/tests/halo2_circuit.rs

```diff
@@ -159,35 +197,62 @@ fn test_empty_sector_update_circuit_1kib_halo2() {
     // Halo2 keygen, proving, and verifying are slow and consume a lot of memory, thus we only test
     // those for a small sector size circuit (the halo2 compound proof tests will run the halo2
     // prover and verifier for larger sector sizes).
+    init_logger();
```

Comment:
Maybe move logger initialization to the top of the inner test function so logging can be disabled via a single comment?